wizard_builder/tests/test_sites.py

```diff
@@ -1,10 +1,11 @@
 from mock import patch
-from wizard_builder.models import PageBase, QuestionPage, SingleLineText
 
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.test import TestCase, override_settings
 
+from ..models import PageBase, QuestionPage, SingleLineText
```

Comment:
I ran into an IMMENSELY frustrating issue where the lines in the tests that did `from wizard_builder import ...` were actually importing an installed copy of wizard builder, which was breaking all of my subprocess calls